androidApp/src/main/java/co/nimblehq/avishek/phong/kmmic/android/ui/screen/common/Background.kt

```diff
@@ -0,0 +1,18 @@
+package co.nimblehq.avishek.phong.kmmic.android.ui.screen.common
+
+import androidx.compose.foundation.Image
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import co.nimblehq.avishek.phong.kmmic.android.R
+
+@Composable
+fun Background(modifier: Modifier) {
```

Comment:
Modifier should be an optional parameter
```suggestion
fun Background(modifier: Modifier = Modifier) {
```

Reply:
Added in acd211abce503054711d0c24171bd83cc8117768